addons/website/static/src/scss/website.scss

```diff
@@ -411,6 +449,27 @@ h1, h2, h3, h4, h5, h6 {
         }
     }
 }
+@for $index from 1 through 4 {
+    .display-#{$index} {
```

Comment:
Spec was made under BS4, where there were no display-5 and 6. Once we saw that they added those, we did not see the need (we already had a hard time to have 1 -> 4 be accepted, FP never liked the fact there were 10 levels of sizes... and he's right on that).

Do you really think there is a need for 12 level stack of sizes? In the end, it's only *default BS5* which defines 6 display-x classes, it could be only 4 (there is variable controlling that).

Reply:
> Do you really think there is a need for 12 level stack of sizes?

Nope, you can design a great website with, for example, just **one** font-family and `display-[1/4]` classes  😉👍 

My point is that, since we allow users to heavily customise display-x (font, size, margin) , they may decide to use these classes for all snippets' titles, while keeping a different design for "normal titles".

In this case you need 5 and 6 too 'cause the first 4 are normally way bigger than the default font size


![image](https://github.com/odoo/odoo/assets/8558351/acea0126-79bf-47d3-bef8-35726d9c769f)

>  FP never liked the fact there were 10 levels of sizes... and he's right on that).

Yes, I do agree as well. It's more an UI problem IMO... I don't have a solution but maybe we can think about it... 🤔

Reply:
Mmh, I see what you mean. Would not we better off compromising and reducing the size of display-4 by default then (and the other 3 big ones accordingly)? So that display-4 is more of the size of the display-5 in your example? I understand your point that would not met your point entirely which is "display-x" could be used to have "another set of tittle style" but that was really not the goal in the first place. The original goal was really about sizing: instead of a h1 with an hardcoded inline style of 62px in snippets, let's use display classes for that. As we cannot just say "all h1 are 62px tall by default" as we do want small h1 in normal text.